core/chains/evm/client/send_only_node.go

```diff
@@ -77,8 +78,15 @@ func (s *sendOnlyNode) Start(startCtx context.Context) error {
 	s.rpc = rpc
 	s.geth = ethclient.NewClient(rpc)
 
-	if err := s.verify(startCtx); err != nil {
-		return errors.Wrap(err, "failed to verify sendonly node")
+	if id, err := s.getChainID(startCtx); err != nil {
+		s.log.Warn("sendonly rpc ChainID verification skipped", "err", err)
```

Comment:
Just warn for _any_ error (which now includes zero/empty chain ID).

Reply:
Even if all the endpoints are compliant and support the chaind ID rpc method, we still can't block startup when one of them is unreachable.